data/i18n/pt/pt.toml

```diff
@@ -0,0 +1,225 @@
+ # i18n strings for the Portuguese (main) site.
```

Comment:
Hi @sftim .
Is this the correct way to include `pt` and `pt-br` localization string files?
Or is it: `data/i18n/pt-br.toml` 
The theme has a shallow hierarchy (listing all localization string files under `i18n`).